interfaces/builtin/multipass.go

```diff
@@ -0,0 +1,50 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const multipassSummary = `allows access to the Multipass socket`
+
+const multipassBaseDeclarationSlots = `
+  multipass:
+    allow-installation: false
+    deny-connection: true
+    deny-auto-connection: true
+`
+
+const multipassConnectedPlugAppArmor = `
+# Description: allow access to the Multipass daemon socket.
+
+/run/multipass_socket rw,
```

Comment:
Related to the above question on the base declaration and the providing slot-- is there any reason this is here rather than SNAP_COMMON, a content slot  or similar? You mentioned that the socket protocol is not stable, maybe we don't need to expose the socket in a place for others to use?

Reply:
What is the response to Jamie's question?

Reply:
I've relocated the socket to $SNAP_COMMON, so this is unnecessary. That meant a client-specific interface became unecessary, so was removed